pkg/system/stat_unix_test.go

```diff
@@ -31,7 +31,7 @@ func TestFromStatT(t *testing.T) {
 	if stat.Gid != s.GID() {
 		t.Fatal("got invalid gid")
 	}
-	if stat.Rdev != s.Rdev() {
+	if uint64(stat.Rdev) != s.Rdev() {
```

Comment:
Looks like you need to add a `//nolint:`

```
[2020-03-13T02:31:15.616Z] pkg/system/stat_unix_test.go:34:11: unnecessary conversion (unconvert)
[2020-03-13T02:31:15.616Z] 	if uint64(stat.Rdev) != s.Rdev() {
```

This will probably work (the comment helps understand why we're doing the conversion);

```suggestion
	//nolint:unconvert // conversion needed to fix mismatch types on mips64el
	if uint64(stat.Rdev) != s.Rdev() {
```

Reply:
I got it, Thank you very much!